jsonschema2pojo-core/src/test/java/org/jsonschema2pojo/rules/AdditionalPropertiesRuleTest.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright © 2010-2020 Nokia
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jsonschema2pojo.rules;
+
+import static org.junit.Assert.assertEquals;
+
+import org.jsonschema2pojo.Schema;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.sun.codemodel.JClass;
+import com.sun.codemodel.JClassAlreadyExistsException;
+import com.sun.codemodel.JCodeModel;
+import com.sun.codemodel.JDefinedClass;
+import com.sun.codemodel.JMethod;
+import com.sun.codemodel.JType;
+
+public class AdditionalPropertiesRuleTest {
+
+    static final ObjectMapper mapper = new ObjectMapper();
+    RuleFactory factory;
+    AdditionalPropertiesRule rule;
+
+    @Before
+    public void setUp() throws JClassAlreadyExistsException {
```

Comment:
Nit:

- `static final ObjectMapper mapper = new ObjectMapper();` doesn't need to be `static`
- doesn't look like `throws JClassAlreadyExistsException` is needed
- `rule` could be declared within method ie;
```java
    @Test
    public void testSchemaWithNullURI() throws Exception {
        final AdditionalPropertiesRule rule = new AdditionalPropertiesRule(new RuleFactory());
```